trickster/endpoints/internal.py

```diff
@@ -111,7 +111,7 @@ def create_route_response(
             route.validate_new_response(new_response)
             route.responses.append(new_response)
             return route
-        except ValidationError as e:
+        except ValidationError as e:  # pragma: no cover
```

Comment:
Why "no cover" here? Was it difficult to simulate invalid payload?

Reply:
It somewhat is, when trying to add an invalid response, you'll get a different exception in a different place and I can't gen into that block of code.

```
new_response = {'status_code': http.HTTPStatus.OK, 'body': {'user_id': 'asdfaf', 'some': 'field'}}
...
>       raise ValueError(f'Response "{response.id}" doesn\'t match ony of the configured validators.')
E       ValueError: Response "085e231a-66a7-4005-9750-2b6a0f22cc6a" doesn't match ony of the configured validators.
```